.travis.yml

```diff
@@ -9,10 +9,19 @@ services:
 
 install:
   - docker --version
+  - pip install coveralls
+  - pip install pytest-cov
```

Comment:
as you guys pointed out, the two lines above:

    - pip install coveralls
    - pip install pytest-cov

should be removed from here  and  Python deps should be added to the Pipfile. In the `script` section below you `source setup_dev_env.sh` which in turn starts a `pipenv shell`, so all deps will be available in the Travis environment as well as when we run it locally. Thanks!!!